ocd_backend/transformers/organizations.py

```diff
@@ -0,0 +1,104 @@
+    }
+
+    object_model = TopLevelOrganization(original_item['systemid'],
+                                        self.source_definition,
+                                        **source_defaults)
+    object_model.entity = entity
+    object_model.classification = u'Municipality'
+    object_model.collection = self.source_definition['key']
+    object_model.name = ' '.join([self.source_definition.get('municipality_prefix', ''), unicode(original_item['naam'])])
+    object_model.description = original_item['omvatplaats']
+    # object_model.contact_details = transform_contact_details(original_item['contact'])
+
+    object_model.save()
+    return object_model
+
+
+@celery_app.task(bind=True, base=BaseTransformer, autoretry_for=(Exception,), retry_backoff=True)
+def province_organization_item(self, content_type, raw_item, entity, source_item, **kwargs):
+    original_item = self.deserialize_item(content_type, raw_item)
+    self.source_definition = kwargs['source_definition']
+
+    source_defaults = {
+        'source': self.source_definition['key'],
+        'supplier': 'allmanak',
+        'collection': 'province',
+    }
+
+    object_model = TopLevelOrganization(original_item['systemid'],
+                                        self.source_definition,
+                                        **source_defaults)
+    object_model.entity = entity
+    object_model.classification = u'Province'
+    object_model.collection = self.source_definition['key']
+    object_model.name = unicode(original_item['naam'])
+    object_model.description = original_item['omvatplaats']
+    # object_model.contact_details = transform_contact_details(original_item['contact'])
+
+    object_model.save()
+    return object_model
+
+
+@celery_app.task(bind=True, base=BaseTransformer, autoretry_for=(Exception,), retry_backoff=True)
+def party_item(self, content_type, raw_item, entity, source_item, **kwargs):
+    original_item = self.deserialize_item(content_type, raw_item)
+    self.source_definition = kwargs['source_definition']
+
+    source_defaults = {
+        'source': self.source_definition['key'],
+        'supplier': 'allmanak',
+        'collection': 'party',
+    }
+
+    # When the Allmanak implements parties as entities, the entity ID should be used
+    object_model = Organization(original_item['partij'],
+                                self.source_definition,
+                                **source_defaults)
+    object_model.entity = entity
+    object_model.has_organization_name = TopLevelOrganization(self.source_definition['allmanak_id'],
+                                                              source=self.source_definition['key'],
+                                                              supplier='allmanak',
+                                                              collection=self.source_definition['source_type'])
+    object_model.collection = self.source_definition['key'] + '-' + original_item['partij']
```

Comment:
It's necessary here because a party's IRI is constructed from information from the Allmanak, which is not available here. This is actually the only merge in use right now.